apps/zero-runtime-vite-app/src/App.tsx

```diff
@@ -57,11 +58,7 @@ export function App({ isRed }: AppProps) {
   const [isHorizontal, setIsHorizontal] = React.useState(true);
 
   return (
-    <div
-      className={css`
-        color: red;
-      `}
-    >
```

Comment:
I was looking into doing something like this pattern in the grid, is the `css` template string syntax going away?

Reply:
Zero-runtime still supports `css` as far as I know.

Reply:
Correct. I just removed it from the demo since it was not a good showcase of its usage.